### DISCUSSION TITLE
Unable to determine diffs

### DISCUSSION THREAD

**Discussion by IdRatherStand** [networktocode/diffsync#230] [Q&A]
Hey - I'm looking to rewrite a large portion of our sync scripts using diffsync, however I'm struggling to get an initial pass working. We are looking to perform syncronisation of prefixes between nautobot -> panorama. 

I've created my prefixes class:
```
class dPrefix(DiffSyncModel):
    """Initial Prefix Model"""

    _modelname = "prefix"
    _identifiers = ("prefix",)
    _attributes = ("status","site","role","vlan_id","classification")

    prefix: str
    status: Optional[str]
    site: Optional[str]
    role: Optional[str]
    vlan_id: Optional[int]
    classification: Optional[str]
```

Once I load my models, I can inspect them using .dict() and can see we return a dict, with key "prefix", under this we then have our individual prefixes with attributes, as seen below:

'10.X.Y.Z/24':{'classification': 'A_Network', 'prefix': '10.X.Y.Z/24', 'site': 'A_Site', 'status': 'active'}

From here, whichever way I then run nautobot.diff_to(panorama), diffsync just immediately reports there are 0 changes, 0 skips etc. Am I fundamentally misunderstanding how I need to write / use the model / adapters?
 
It looks as though it should be working to me, I can clearly see there are differences in the datasets.

**Comment by Kircheneer** [comment]
Can you share your full code? Just with this its hard to tell where it might go wrong.
